combat-trainer.lic

```diff
@@ -371,6 +371,12 @@ class LootProcess
 
     @rituals = get_data('spells').rituals
     echo("  @rituals: #{@rituals}") if $debug_mode_ct
+    
+    @redeemed = settings.necro_redeemed
+    echo("  @redeemed: #{@redeemed}") if $debug_mode_ct
+
+    @force_rituals = settings.necro_force_rituals
```

Comment:
Allows user to force rituals.  Handy when farming materials or pushing hidden butcher skill.  relates to line 579